packages/client/src/home/index.tsx

```diff
@@ -11,13 +11,16 @@ const Home = () => {
     const [isLoading, setLoading] = useState<boolean>(false);
 
     useEffect(() => {
+        const headers = new Headers();
+        headers.append('Content-Type', 'application/json');
```

Comment:
@Hvinder The root cause was headers not being set correctly. So express was not able to read the JSON body correctly.